Python/bytecodes.c

```diff
@@ -1364,6 +1393,9 @@ dummy_func(
             }
         }
 
+        macro(LOAD_CLASSDEREF) = _LOAD_CLASSDEREF_INTRO + _LOAD_CLASSDEREF_COMMON;
```

Comment:
Possibly, but I'd prefer to avoid touching the implementation of existing opcodes at this point. Do you think it's important to replace LOAD_CLASSDEREF here? I think we can reconsider this when we implement PEP 649 for 3.13.

Reply:
Yeah I think it's fine to consider consolidation of redundant opcodes separately later.